libs/blocks/quiz-results/quiz-results.js

```diff
@@ -0,0 +1,101 @@
+import { createTag } from '../../utils/utils.js';
+import { getMetadata, handleStyle } from '../section-metadata/section-metadata.js';
+
+export const LOADING_ERROR = 'Could not load quiz results:';
```

Comment:
Will this need to be localized?

Reply:
Hi @rgclayton. We do have a future story to localize the thing. https://jira.corp.adobe.com/browse/MWPW-136320

Reply:
Actually, no, not this particular message. This message is just useful when the debug flag is set. If there's a problem in quiz-results, we redirect to the uber creative cloud landing page anyway, and log a message in lana. The story I mentioned refers to other messages in the quiz block, not quiz-results.